docs/index.html

```diff
@@ -271,6 +271,8 @@ <h6>Install</h6>
       <pre><code class="hljs console">  npm install frappe-charts</code></pre>
       <p>And include it in your project</p>
       <pre><code class="hljs javascript">  import { Chart } from "frappe-charts"</code></pre>
+      <p>(for ES6+ import the ESModule from the dist folder)</p>
+      <pre><code class="hljs javascript">  import { Chart } from "frappe-charts/dist/frappe-charts.esm.js"</code></pre>
```

Comment:
Any reason for it to not point to the minified file?

Reply:
Nope 👍